### DISCUSSION TITLE
cannot get .su2 mesh files for .cgns mesh input

### DISCUSSION THREAD

**Discussion by LJ1937** [su2code/SU2#2147] [Q&A]
https://su2code.github.io/tutorials/Inviscid_Wedge/

For this compressible flow,Inviscid Supersonic Wedge. I noticed that in the cfg file,
```
% Mesh input file
MESH_FILENAME= mesh_wedge_inv.cgns
%
% Mesh input file format (SU2, CGNS, NETCDF_ASCII)
MESH_FORMAT= CGNS
%
% Mesh output file
MESH_OUT_FILENAME= mesh_out.su2
```

But after running successfully, I didnot get the "mesh_out.su2" file,why?
![image](https://github.com/su2code/SU2/assets/62540357/c2f57d31-4e90-4e5c-8de2-85ce7cfb0765)

**Comment by bigfooted** [comment]
This output mesh is not written as part of the CFD run. We use SU2_DEF instead of SU2_CFD for mesh deformation operations. If you just want to convert the mesh to .su2, use DV_KIND=NO_DEFORMATION and run SU2_DEF.
